WebApiBackend/WebApiBackend/EF/ChoresRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using WebApiBackend.Model;
+
+namespace WebApiBackend.EF
+{
+    public class ChoresRepository : EfRepository<Chores, FlatManagementContext>
+    {
+        public ChoresRepository(FlatManagementContext context) : base(context)
```

Comment:
Looking at the existing code, it seems we might need to add Chores as a field to the FlatManagementContext.cs file. It seems like this might add the table to the whole datbase context

Reply:
Good catch. Please review changes. Looking at the other examples, I don't think Chores needs any OnModelCreating implementation of its own.

Reply:
Yep I think you're right. It looks good to me now